numpyro/handlers.py

```diff
@@ -625,25 +619,22 @@ class scope(Messenger):
     :param fn: Python callable with NumPyro primitives.
     :param str prefix: a string to prepend to sample names
     :param str divider: a string to join the prefix and sample name; default to `'/'`
+    :param bool skip_param: whether to apply scope for `param` sites. Defaults to True.
     """
 
-    def __init__(self, fn=None, prefix="", divider="/"):
+    def __init__(self, fn=None, prefix="", divider="/", *, skip_param=True):
```

Comment:
Not sure which argument we need to add here. Probably we can replace this with `optional_types` which allows users to specify additional types to apply scope to, e.g. `params` or `plate`.